cmd/dbtool.cpp

```diff
@@ -122,7 +122,7 @@ struct dbFreeInfo {
 struct db_options_t {
     std::string datadir{silkworm::db::default_path()};  // Where data file is located
     std::string mapsize_str{};                          // Provided map_size literal
-    size_t mapsize{0};                                  // Computed map size
+    uint64_t mapsize{0};                                // Computed map size
```

Comment:
Please leave it as size_t because otherwise it fails to compile for me on macOS.